### DISCUSSION TITLE
Linux version of ioSender?

### DISCUSSION THREAD

**Discussion by jdeline** [terjeio/ioSender#170] [Q&A]
Terjeio, any chance there will be a linux version of ioSender?  I'd like to run my mill with my laptop, which is currently running Linux Mint.  I'd rather not go the Windows route via VM because I would still have to purchase a Windows license, and I'm not really fond of running under VMs anyway.

**Comment by terjeio** [comment]
> Terjeio, any chance there will be a linux version of ioSender?

Not until [Mono](https://www.mono-project.com/) get support for WPF or somebody ports it to a different language and framework.
A quick search and I found [this discussion](https://github.com/dotnet/wpf/issues/48) that may shed some light on the issues involved.

**Reply by kaganar** [reply]
I'm new here and necroing an old issue, however...

@terjeio Would you consider adopting Linux support if a fork successfully used something like Avalonia and demonstrated a sufficient amount of viability? Or would there be other barriers that would need resolution before adoption?
